NorthstarDLL/httprequesthandler.h

```diff
@@ -0,0 +1,132 @@
+#pragma once
+
+#include <string>
```

Comment:
These are included in the PCH file.

Reply:
Resolved on new branch, will re-make PR.